clap_mangen/src/render.rs

```diff
@@ -9,9 +9,10 @@ pub(crate) fn subcommand_heading(cmd: &clap::Command) -> &str {
 }
 
 pub(crate) fn about(roff: &mut Roff, cmd: &clap::Command) {
+    let name = cmd.get_display_name().unwrap_or_else(|| cmd.get_name()).to_string();
     let s = match cmd.get_about().or_else(|| cmd.get_long_about()) {
-        Some(about) => format!("{} - {}", cmd.get_name(), about),
-        None => cmd.get_name().to_string(),
+        Some(about) => format!("{} - {}", name, about),
+        None => name,
```

Comment:
This has a shorter line width (`cargo fmt` is happy then) and omits a copy (which is basically pointless for a probably compile time job, but I still prefer better code especially when it's easy).

```diff
-    let name = cmd.get_display_name().unwrap_or_else(|| cmd.get_name()).to_string();
+    let name = cmd.get_display_name().unwrap_or_else(|| cmd.get_name());
     let s = match cmd.get_about().or_else(|| cmd.get_long_about()) {
         Some(about) => format!("{} - {}", name, about),
-        None => name,
+        None => name.to_owned(),
```